en/editor/toolbar/index.md

```diff
@@ -1,127 +1,127 @@
-点击主窗口左上角工具栏第三个按钮，或在编辑场景时按下 **R** 快捷键，即可切换使用 **缩放变换工具**。
+Click the third button on the toolbar in the upper left corner of the main window, or press the **R** shortcut while editing a scene to switch to the **Scale Gizmo**.
 
 ![scale tool](./img/scale-tool.png)
 
-缩放变换工具由三个头部是红绿蓝正方体的坐标轴以及中心一个灰色正方体组成。当鼠标悬浮在任一正方体上时，显示为黄色，点击即可选中并拖动：
+The scale tool consists of three axes with red, green and blue squares at the head and a gray square at the center. When the mouse is hovered over any of the squares, it appears yellow and can be selected and dragged by clicking on.
 
-- 按住红色/绿色/蓝色正方体拖拽鼠标，将分别在 X、Y、Z 轴方向上缩放节点；
-- 按住灰色正方体拖拽鼠标，将同时在 X、Y、Z 轴方向上缩放节点。
+- Holding the red/green/blue square and dragging the mouse will scale the node in the X, Y and Z axis directions respectively.
+- Dragging the mouse on a gray square will scale the node in X, Y, and Z directions at the same time.
 
 ![scale gizmo](./img/scale-gizmo.png)
 
-### 矩形变换工具
+### Rect Gizmo
 
-点击主窗口左上角工具栏第四个按钮，或在编辑场景时按下 **T** 快捷键，即可切换使用 **矩形变换工具**。需要注意的是，矩形变换工具只适用于 UI 节点。
+Click the fourth button on the toolbar in the upper left corner of the main window, or press the **T** shortcut while editing a scene to switch to the **Rect Gizmo**. Note that the Rect Gizmo is only available for UI nodes.
 
 ![rect tool](./img/rect-tool.png)
 
-矩形变换工具由四个顶点控制点、四个边控制点、一个中心控制点组成。
+The Rect Gizmo consists of four vertex control points, four edge control points, and one center control point.
 
 ![rect gizmo](./img/rect-gizmo.png)
 
-矩形变换工具激活时：
-- 拖拽控制手柄的任一顶点控制点，可以在保持对角顶点位置不变的情况下，同时修改 UI 节点的 `Position` 属性和 UITransform 组件中的 `ContentSize` 属性。
-- 拖拽控制手柄的任一边控制点，可以在保持对边位置不变的情况下，修改 UI 节点的 `Position`（`X` 或 `Y` 属性）和 UITransform 组件中的 `ContentSize` 属性（`width` 或 `height` 属性）。
-- 拖拽控制手柄的中心控制点，可以在 UI 节点的尺寸大小不变的情况下，同时修改 UI 节点的 `Position` 属性和 UITransform 组件中的 `AnchorPoint`（锚点）属性。
+When the Rect Gizmo is active:
+- Dragging either vertex control point of the control handle can modify both the `Position` property of the UI node and the `ContentSize` property in the UITransform component while keeping the position of the diagonal vertices unchanged.
+- Dragging either side of the control handle modifies both the `Position` (`X` or `Y` property) of the UI node and the `ContentSize` property (`width` or `height` property) of the UITransform component, while keeping the position of the opposite side unchanged.
+- Dragging the center control point of a control handle allows to modify both the `Position` property of the UI node and the `AnchorPoint` property of the UITransform component while keeping the size of the UI node unchanged.
 
-在 UI 元素的排版中，经常会需要使用 **矩形变换工具** 直接精确控制节点四条边的位置和长度。而对于必须保持原始图片宽高比的图像元素，通常不会使用矩形变换工具来调整尺寸。
+In the layout of UI elements, it is often necessary to use the **Rect Gizmo** to directly control the position and length of the node's four edges precisely. For image elements where the original image aspect ratio must be maintained, the Rect Gizmo is not usually used to resize.
 
-## Gizmo 基准点设置
+## Gizmo Datum Setting
```

Comment:
what is a datum?

Reply:
Datum mark, datum point, or control point? I don't know which one is right.

Reply:
control point, I understand this.